extensions/amp-lightbox-gallery/0.1/service/lightbox-manager-impl.js

```diff
@@ -38,18 +38,13 @@ import {map} from '../../../../src/core/types/object';
 import {srcsetFromElement, srcsetFromSrc} from '../../../../src/srcset';
 import {toArray} from '../../../../src/core/types/array';
 
-const LIGHTBOX_ELIGIBLE_TAGS = {
-  'AMP-IMG': true,
-};
+const LIGHTBOX_ELIGIBLE_TAGS = new Set(['AMP-IMG', 'IMG']);
 
-export const ELIGIBLE_TAP_TAGS = {
-  'AMP-IMG': true,
-};
+// eslint-disable-next-line local/no-export-side-effect
+export const ELIGIBLE_TAP_TAGS = new Set(['AMP-IMG']);
 
-export const VIDEO_TAGS = {
-  'AMP-YOUTUBE': true,
-  'AMP-VIDEO': true,
-};
+// eslint-disable-next-line local/no-export-side-effect
+export const VIDEO_TAGS = new Set(['AMP-YOUTUBE', 'AMP-VIDEO']);
```

Comment:
I believe I was the one who suggested a set over an object here a while ago, but have since learned ways non-static modules can be problematic for compiler/bento. Since this is a 0.1 I'm guessing it's fine, but consider double-checking with Justin and/or Jake that it's okay here

Reply:
@samouri or @jridgewell what's the issue with exporting a `Set`?

Reply:
I was unsure whether or not module-level `Set` could be DCEd. If the compiler weren't smart enough to realize that the `Set` constructor was side-effect free, then it would be force to leave it in.

I [just confirmed](https://closure-compiler.appspot.com/home#code%3D%252F%252F%2520%253D%253DClosureCompiler%253D%253D%250A%252F%252F%2520%2540compilation_level%2520ADVANCED_OPTIMIZATIONS%250A%252F%252F%2520%2540output_file_name%2520default.js%250A%252F%252F%2520%253D%253D%252FClosureCompiler%253D%253D%250A%250A%250Aconst%2520a%2520%253D%2520new%2520Set()%253B%250Aconst%2520b%2520%253D%2520%2522hello%2520world%2522%253B%250Aconst%2520c%2520%253D%2520%2522hello%2520world%2522%253B%250A%250Aconsole.log(c)%253B) that instantiating Set in module level can be DCEd by Closure.